doc/release-notes-19219.md

```diff
@@ -0,0 +1,20 @@
+#### Change in automatic banning
+
+Automatic banning of peers for bad behavior has been slightly altered:
```

Comment:
Since they're not really bans, let's not call them bans in docs?

Reply:
I think this is improved now with @jnewbery's changes.